sound/soc/sof/trace.c

```diff
@@ -154,15 +163,24 @@ int snd_sof_init_trace_ipc(struct snd_sof_dev *sdev)
 	if (sdev->dtrace_is_enabled)
 		return -EINVAL;
 
+	/* update init_offset, and reinitial offset to 0 */
```

Comment:
reset offset?

Reply:
it's not reset to the same value at each resume, so I am using reset...

Reply:
Okay, changing it to 're-initialize'.